infra/app/database/outputs.tf

```diff
@@ -1,3 +1,9 @@
 output "role_manager_function_name" {
   value = module.database.role_manager_function_name
 }
+
+output "db_config" {
```

Comment:
Would love feedback on a more specific name here, we use `db_config` or various permutations of that a lot in the codebase which can make it hard to follow

Reply:
Maybe, `role_manager_db_config`?

Reply:
this output is unnecessary. we should be able to pull the db config from the app-config module via

```
terraform -chdir="infra/$APP_NAME/app-config" output -json environment_configs | jq -r ... 
```

see [this line](https://github.com/navapbc/template-infra/blob/b8ab422b4c905138c9572a4abf5a2cc9d44fb1a3/bin/run-database-migrations.sh#L40) as an example

Reply:
Does that work prior to running e.g., `make infra-configure-app-service APP_NAME=app ENVIRONMENT=dev`? `make infra-update-app-database-roles APP_NAME=app ENVIRONMENT=dev` is run before that step in the (current) set up instructions

Reply:
`project-config` and `app-config` are static, which make them available before any resources have been created

Reply:
you'l want to add these lines at the top of your script:

```
terraform -chdir="infra/$APP_NAME/app-config" init > /dev/null
terraform -chdir="infra/$APP_NAME/app-config" apply -auto-approve > /dev/null
```

which ensures the app config has been initialized